cms/static/sass/views/_container.scss

```diff
@@ -55,6 +55,14 @@
             }
           }
         }
+
+        .container-unit-access {
+          @include font-size(14);
+          line-height: 1.5;
+          white-space: normal;
+          color: $black;
```

Comment:
@roderickmorales Is this the color you want for the text? It looks a bit "black". :)

Perhaps we should be consistent with the color that is used for component access? 

![image](https://user-images.githubusercontent.com/484484/27609618-699a5f4a-5b59-11e7-934f-e3801eb6135e.png)